configs/animal_2d_keypoint/topdown_heatmap/README.md

```diff
@@ -1,7 +1,53 @@
 # Top-down heatmap-based pose estimation
 
-Top-down methods divide the task into two stages: object detection and pose estimation.
+Top-down methods divide the task into two stages: object detection, followed by single-object pose estimation given object bounding boxes Instead of estimating keypoint coordinates directly, the pose estimator will produce heatmaps which represent the
+likelihood of being a keypoint, following the paradigm introduced in [Simple Baselines for Human Pose Estimation and Tracking](http://openaccess.thecvf.com/content_ECCV_2018/html/Bin_Xiao_Simple_Baselines_for_ECCV_2018_paper.html).
 
-They perform object detection first, followed by single-object pose estimation given object bounding boxes.
-Instead of estimating keypoint coordinates directly, the pose estimator will produce heatmaps which represent the
-likelihood of being a keypoint.
+<div align=center>
+<img src="https://user-images.githubusercontent.com/15977946/146522977-5f355832-e9c1-442f-a34f-9d24fb0aefa8.png" height=400>
+</div>
+
+## Results and Models
+
+### Animal-Pose Dataset
+
+Results on AnimalPose validation set (1117 instances)
+
+|   Model    | Input Size |  AP   |  AR   |                   Details and Download                    |
+| :--------: | :--------: | :---: | :---: | :-------------------------------------------------------: |
+| HRNet-w32  |  256x256   | 0.740 | 0.780 |  [hrnet_animalpose.md](./animalpose/hrnet_animalpose.md)  |
+| HRNet-w48  |  256x256   | 0.738 | 0.778 |  [hrnet_animalpose.md](./animalpose/hrnet_animalpose.md)  |
+| ResNet-152 |  256x256   | 0.704 | 0.748 | [resnet_animalpose.md](./animalpose/resnet_animalpose.md) |
+| ResNet-101 |  256x256   | 0.696 | 0.736 | [resnet_animalpose.md](./animalpose/resnet_animalpose.md) |
+| ResNet-50  |  256x256   | 0.691 | 0.736 | [resnet_animalpose.md](./animalpose/resnet_animalpose.md) |
+
+### AP-10K Dataset
+
+Results on AP-10K validation set
+
+|   Model    | Input Size |  AP   |            Details and Download            |
+| :--------: | :--------: | :---: | :----------------------------------------: |
+| HRNet-w48  |  256x256   | 0.728 |  [hrnet_ap10k.md](./ap10k/hrnet_ap10k.md)  |
+| HRNet-w32  |  256x256   | 0.722 |  [hrnet_ap10k.md](./ap10k/hrnet_ap10k.md)  |
+| ResNet-101 |  256x256   | 0.681 | [resnet_ap10k.md](./ap10k/resnet_ap10k.md) |
+| ResNet-50  |  256x256   | 0.680 | [resnet_ap10k.md](./ap10k/resnet_ap10k.md) |
+
+### Desert Locust Dataset
+
+Results on Desert Locust test set
+
+|   Model    | Input Size | PCK@0.2 |  AUC  | EPE  |             Details and Download              |
```

Comment:
It's unnecessary to show PCK in this table since the values are all 1.000. Same for the table of Grévy’s Z.ebra dataset.

Reply:
Done.